docs/plugins/cloud-gce.asciidoc

```diff
@@ -1,54 +1,53 @@
-Google Compute Engine Cloud Plugin for Elasticsearch
-====================================================
+[[cloud-gce]]
+=== GCE Cloud Plugin
 
-The GCE Cloud plugin allows to use GCE API for the unicast discovery mechanism.
+The Google Compute Engine Cloud plugin uses the GCE API for unicast discovery.
 
-In order to install the plugin, run: 
+[[cloud-gce-install]]
+[float]
+==== Installation
 
-```sh
-bin/plugin install elasticsearch/elasticsearch-cloud-gce/2.5.0
-```
+This plugin can be installed using the plugin manager:
 
-You need to install a version matching your Elasticsearch version:
+[source,sh]
+----------------------------------------------------------------
+bin/plugin install elasticsearch-cloud-gce
```

Comment:
Is this right? It doesn't work on ES 1.6, but I don't have a 2.0-SNAPSHOT around to test.

I love that we don't need the version support table thing anymore, but perhaps we can replace it with a NOTE: that says something to the effect of "If you are running a version of Elasticsearch < 2.0, please go HERE to find the compatible version".